SDKTest/EventTests.swift

```diff
@@ -5,133 +5,138 @@
 //  Created by Clayton Selby on 8/19/21.
 //
 
-import XCTest
 @testable import NeuroID
+import XCTest
 
 class EventTests: XCTestCase {
-    
     let clientKey = "key_live_vtotrandom_form_mobilesandbox"
     let userId = "form_mobilesandbox"
     
     override func setUpWithError() throws {
         NeuroID.configure(clientKey: clientKey)
     }
     
-    func testSetScreeName(){
+    override func setUp() {
+        // Clear out the DataStore Events after each test
+        DataStore.events = []
+    }
```

Comment:
@cometman - This is the setup function that will clear out the `DataStore` after each test because it was building up events throughout the tests and causing unintended side effects